README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+
+
+> # "Anyone Can Cook""
+> ## - Chef Gusteau 
+
+![Chef Gusteau and Remy](https://user-images.githubusercontent.com/65682542/85363397-749ac800-b4e6-11ea-8ffb-5eab32255b88.jpg)
```

Comment:
Nice choice! And how appropriate for this course

![](https://media.giphy.com/media/8y8NVqFqhTQVa/giphy.gif)